recipe/meta.yaml

```diff
@@ -12,6 +12,8 @@ source:
   sha256: {{ sha256 }}
 
 build:
+  skip: True  # [py<36]
+  noarch: python
```

Comment:
@katietz Is it good to switch to noarch like upstream conda-forge or we should wait for the major version bump?